### PR TITLE
test: fix lwc hover tests in autobuilds

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Fixes LWC hover tests from failing in autobuilds due to language client not being ready yet.

### What issues does this PR fix or reference?

@W-8583127@

### CODE REVIEW

File: packages/salesforcedx-vscode-lwc/src/languageClient/index.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { Uri, workspace } from 'vscode';
```
Comment (lcampos) [inline packages/salesforcedx-vscode-lwc/src/languageClient/index.ts:1]:
```suggestion
/*
 * Copyright (c) 2020, salesforce.com, inc.
 * All rights reserved.
 * Licensed under the BSD 3-Clause license.
 * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
import { Uri, workspace } from 'vscode';
```

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts [hunk 2]
```diff
@@ -25,11 +27,33 @@ describe('LWC Hovers', () => {
...
+  });
+
   afterEach(async () => {
     await commands.executeCommand('workbench.action.closeActiveEditor');
   });
 
+  after(() => client.stop());
+
   it('Should provide additional details when hovering over a LWC tag', async () => {
+    await client.onReady();
```
Comment (lcampos) [inline packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts:56]: Are there other tests we should update (later) with these changes ?

File: packages/salesforcedx-vscode-lwc/src/index.ts [hunk 3]
```diff
@@ -107,7 +102,18 @@ export async function activate(context: ExtensionContext) {
   console.log('WorkspaceType detected: ' + workspaceType);
 
   // Start the LWC Language Server
-  startLWCLanguageServer(context);
+  const client = createLanguageClient(
+    context.asAbsolutePath(
```
Comment (lcampos) [inline packages/salesforcedx-vscode-lwc/src/index.ts:106]: Have we tested this with the generated vsix ? It looks correct but we should confirm

**Review by lcampos — APPROVED** [review]

File: packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts (hunk 2 above)
Comment (brpowell) [reply packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts]: Possibly the intellisense one, but not sure if its worth changing now or if it becomes a problem. What do you think?

Comment (lcampos) [reply packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts]: I agree in not changing them right now, just asking so we can log a work item and get to them in the next couple of weeks.

Comment (brpowell) [reply packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts]: I'll log something to track it

**Comment by brpowell** [comment]
@lcampos I tested out the visixes from the current build and the language server features seem to be working correctly. I can put in the QA instructions to test with them as well whoever picks it up.

File: packages/salesforcedx-vscode-lwc/src/languageClient/index.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        workspace.createFileSystemWatcher('**/', false, true, false)
+      ]
+    },
+    uriConverters: {
+      code2Protocol: code2ProtocolConverter,
+      protocol2Code: protocol2CodeConverter
+    }
+  };
+
+  // Create the language client and start the client.
```
Comment (brpowell) [inline packages/salesforcedx-vscode-lwc/src/languageClient/index.ts:72]:
```suggestion
```

**Pull request merged**